assets/src/design-system/components/button/button.js

```diff
@@ -61,6 +61,15 @@ const Base = styled.button(
       color: ${theme.colors.interactiveFg.active};
     }
 
+    /* additional specifications for anchors are necessary for wordpress to override common css */
+    ${as === 'a' &&
+    css`
+      &:hover,
+      &:focus {
+        color: ${theme.colors.interactiveFg.active};
+      }
+    `}
+
```

Comment:
🤔 should we be using the `Button` from the design system here? We could style a plain button just like this if the styling needs to be special

Reply:
Maybe i'm missing something? I'd already adjusted the buttons to also be links since there's always some links that are styled in the fashion of buttons. This worked fine in storybook but now that common css is getting injected we have to supersede it.